Library/Source/NativeEngine.cpp

```diff
@@ -184,7 +184,7 @@ namespace babylon
                 auto channelCount = (image.m_format == bimg::TextureFormat::RGB8) ? 3 : 4;
                 auto mipMapCount = static_cast<uint32_t>(std::log2(image.m_width));
                 auto mipmapImageSize = image.m_size;
-                for (auto i = 1; i <= mipMapCount; i++)
+                for (auto i = 1u; i <= mipMapCount; i++)
```

Comment:
nit: This seems a little weird. Should we just make the iterator type size_t or uint32_t or something instead, to match what we tend to do elsewhere?

Reply:
Also line 197.

Reply:
Meh, I started with uint32_t. I have no preference. I'll change it.